azure-pipelines.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       inputs:
         versionSpec: '3.8'
     - bash: |
-        pip install flake8 mypy==0.780
+        pip install flake8 mypy==0.782
```

Comment:
can you remind me why we hardcode the version? Is it to avoid differences with the pre-commits?

Reply:
That and to avoid breaking CI if ever they make a backward incompatible release (not sure if they would, but I guess it's safer).